libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -457,8 +458,13 @@ public void testFilesEntitlementsWithExclusive() {
                 Set.of()
             )
         );
-        assertTrue(iae.getMessage().contains("duplicate/overlapping exclusive paths found in files entitlements:"));
-        assertTrue(iae.getMessage().contains(Strings.format("[test] [%s]]", testPath1.toString())));
+        assertThat(
```

Comment:
Looks like this is flaky/ can be ordered differently? https://gradle-enterprise.elastic.co/s/biqc2jrksfxvu/tests/task/:libs:entitlement:test/details/org.elasticsearch.entitlement.runtime.policy.PolicyManagerTests/testFilesEntitlementsWithExclusive?top-execution=1